metricbeat/module/apache/status/status_test.go

```diff
@@ -225,7 +225,6 @@ func TestHostParser(t *testing.T) {
 		{"", "", "empty host"},
 		{":80", "", "empty host"},
 		{"localhost", "http://localhost/server-status?auto=", ""},
-		{"localhost/ServerStatus", "http://localhost/ServerStatus?auto=", ""},
```

Comment:
This test was failing due to the changes in this PR. I decided to remove it because the intent behind this test seems to be to override the apache server status path, which can be accomplished via the `server_status_path` setting in the apache module (which defaults to `"server-status"`).